tests/gorepo_test.go

```diff
@@ -5,11 +5,19 @@ package tests_test
 import (
 	"os"
 	"os/exec"
+	"strconv"
 	"testing"
 )
 
 // Go repository basic compiler tests, and regression tests for fixed compiler bugs.
 func TestGoRepositoryCompilerTests(t *testing.T) {
+	if b, _ := strconv.ParseBool(os.Getenv("SOURCE_MAP_SUPPORT")); os.Getenv("SOURCE_MAP_SUPPORT") != "" && !b {
+		t.Fatal("Source maps disabled, but required for this test. Use SOURCE_MAP_SUPPORT=true or unset it completely.")
```

Comment:
I've looked in https://github.com/golang/go/blob/master/src/cmd/go/go_test.go to see when skip is used.

It's used to skip individual specific tests when environment is insufficient. For example, skip testing cgo when when cgo is not enabled.

The difference is that `TestGoRepositoryCompilerTests` runs many, many tests. Only one of those actually depends on source-map-support to be enabled. Skipping all of `TestGoRepositoryCompilerTests` just because source-map-support is equivalen to skipping _all_ of `cmd/go` tests if cgo is not enabled. I don't think it's acceptable.

Let's find a better solution...